source/process/release-process.md

```diff
@@ -21,197 +21,177 @@ Pre-work for the current release begins at the code complete date of the previou
     - Update [Team](http://www.mattermost.org/team/) page with new contributors
     - Add list of contributors to the Changelog draft
-3. Dev:
+4. Dev:
     - Prioritize reviewing, updating, and merging of pull requests for current release until there are no more tickets in the [pull request queue](https://github.com/mattermost/mattermost-server/pulls) marked for the current release
 4. PM:
-    - Review all [Severity 1 bugs (data loss or security)](https://mattermost.atlassian.net/secure/IssueNavigator.jspa?mode=hide&requestId=10600) to consider for adding to Hotfix list.
-    - Update documentation:  
-        - Submit Changelog PR
-        - Draft [Mattermost Security Updates](http://about.mattermost.com/security-updates/), but do not post until 14 days after official release
-    - If there are any breaking compatibility changes in the release, open an issue in the [GitLab Omnibus](https://gitlab.com/gitlab-org/omnibus-gitlab) to make sure GitLab is aware. Post a link to the issue in the Release Discussion channel.
-    - PM owner of System Admin features should review changes to `config.json` and confirm they're included in the correct Mattermost Edition.
+    - If there are any breaking compatibility changes in the release, open an issue in the [GitLab Omnibus](https://gitlab.com/gitlab-org/omnibus-gitlab) to make sure GitLab is aware. Post a link to the issue in the Release Discussion channel
+    - PM owner of System Admin features reviews changes to `config.json` to confirm they're included in the correct Mattermost Edition
 5. Docs:
-    - Submit all doc PRs for review
+    - Submit any remaining documentation PRs for product updates in the release
     - Confirm changes to config.json in compatibility section of Changelog are written back to [settings documentation](http://docs.mattermost.com/administration/config-settings.html#configuration-settings)
 6. QA:
     - Confirm all PRs merged into the current release have been tested
-    - Coordinate testing:
-        - Receive testing sign-off from feature area owners (i.e. PM/Dev either signs-off that their area is well tested, or flags potential quality issues that may exist)  
-        - Prioritize updating the RC Testing Spreadsheet to cover any changes or new features, and confirm that known issues are listed in the relevant tests
-        - Assign each area of the release testing spreadsheet to a team member and ensure core team has access permissions
-7. **(Team) Code Complete Meeting (10:00am San Francisco time)**:  
-    - (PM) Leads team review of Changelog
-    - (Release Manager) Walk through each unfinished item of this checklist  
-    - (Dev) Last check of tickets that need to be merged before RC1  
-8. Build:  
+    - Ensure the release testing spreadsheet covers any changes and new features, and confirm known issues are listed in the relevant tests 
+    - Assign each area of the spreadsheet to a team member and give the core team access permissions
```

Comment:
Additional QA item for T-8:
  - Update rctesting and CI server invite links in Release Testing spreadsheet after they've been updated to release branch